src/App.js

```diff
@@ -1,5 +1,34 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { MESSAGES } from "./constants/message.js";
+import { isPlayerInputValid, isTrialInputValid } from "./utils/validation.js";
+import move from "./utils/move.js";
+import printResult from "./utils/printResult.js";
+
 class App {
-  async play() {}
+  async play() {
+    // 게임 시작
+    const playerInput = await MissionUtils.Console.readLineAsync(MESSAGES.START_INPUT);
+    const players = isPlayerInputValid(playerInput);
+    const playersMap = new Map(players.map((player) => [player, 0]));
+
+    const tryInput = await MissionUtils.Console.readLineAsync(MESSAGES.TRY_INPUT);
+    const tryNum = isTrialInputValid(tryInput);
+
+    // 게임 진행
+    MissionUtils.Console.print(MESSAGES.RESULT);
+    for (let i = 0; i < tryNum; i++) {
+      move(playersMap);
+      printResult(playersMap);
+    }
+
+    // 게임 끝
+    const maxLength = Math.max(...Array.from(playersMap.values()));
+    const winner = [];
+    playersMap.forEach((value, key) => {
+      if (value === maxLength) winner.push(key);
+    });
```

Comment:
```suggestion
    const playersArray = [...playersMap.values()];
    const maxLength = Math.max(...playersArray);
    const winner = playersArray.filter((length) => length === maxLength);
```
여기 부분을 `filter`를 사용해서 간략하게 표현할 수 있을 것 같아요~!

Reply:
오오! 아직 저는 메서드에 대한 활용이 부족한 것 같아요ㅠ 감사합니다!

Reply:
음... 근데 제안해주신대로 수정하면 `playersArray`에는 `count`들만 들어있어서 각 자동차의 전진한 횟수만 들어있지 않을까요? 그럼 `winner`에는 최댓값 숫자만 남게 될 것 같아요..! 
저는 `Map`을 사용했기 때문에 `winner`를 얻기 위해서는 `value`와 `key`모두 필요해서 `forEach`를 사용했어요!

Reply:
아 저기 playersArray가 들어갔군요 그러면 말씀해주신대로 playersMap을 fillter하시면 해결될 수 있을 것 같아요!